docs/source/index.rst

```diff
@@ -0,0 +1,22 @@
+Scripting in enDAQ Lab
+======================
+
+Overview
+--------
+
+As of enDAQ Lab 2.0, Lab now provides a built-in scripting environment which enables users to programmatically interact in Python with many of the components of enDAQ Lab. This documentation aims to introduce users to the various scripting tools, the underlying Python API, and some common use cases that one might encounter when using the scripting environment.
```

Comment:
Should this reference to the Lab still be here?

Reply:
nope! removed in f34863e2684b9ee9eddd485954b5e896cdc7e481